packages/experimental/src/inbox-note/action.tsx

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
This component used to contain a lot of extra logic for removing notes, I removed all of that, and made it very dumb... 😆